snapcraft.yaml

```
@@ -1448,7 +1448,7 @@ parts:
 
       # Setup bash completion
       mkdir -p "${CRAFT_PART_INSTALL}/etc/bash_completion.d/"
-      "${CRAFT_PART_INSTALL}/bin/lxc" completion bash > "${CRAFT_PART_INSTALL}/etc/bash_completion.d/snap.lxd.lxc"
+      "${CRAFT_PART_INSTALL}/bin/lxc" completion bash |  sed '/\bcomplete\b/ s/lxc$/lxc lxd.lxc/' | sed 's/requestComp="${words\[0\]} __complete ${args\[\*\]}"/requestComp="\$SNAP\/lxc __complete ${args[*]}"/' > "${CRAFT_PART_INSTALL}/etc/bash_completion.d/snap.lxd.lxc"
```

A single `sed` also works using `; ` separator. An alternative would also be a single `sed` but with 2 edits like `sed -e 's/foo/bar/' -e 's/bla/boo/'`.
```suggestion
      "${CRAFT_PART_INSTALL}/bin/lxc" completion bash | sed '/\bcomplete\b/ s/lxc$/lxc lxd.lxc/; s/requestComp="${words\[0\]} __complete ${args\[\*\]}"/requestComp="\$SNAP\/lxc __complete ${args[*]}"/' 
```

I'd also appreciate a quick comment as to what that edit/mangling achieves or fixes.